### PR TITLE
fix(AAE-11533): Trigger Github prerelease for Helm charts on release candidate tag

### PR DESCRIPTION
**Pull request by igdianov**

### CODE REVIEW

**Review by andrea-ligios — APPROVED** [review]
LGTM!

File: .github/workflows/rc.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  notify:
+    runs-on: ubuntu-latest
+    needs:
+      - prerelease
+    if: always() && failure() && github.event_name == 'push'
+    steps:
+      - name: Slack Notification
+        uses: Alfresco/alfresco-build-tools/.github/actions/send-slack-notification@v1.22.0
+        with:
+          channel-id: 'eng-hxp-studio-be-gh-notifs'
```
Comment (atchertchian) [inline .github/workflows/rc.yml]: will the notif channel be changed when the new one is fully configured?

**Review by atchertchian — APPROVED** [review]

File: .github/workflows/rc.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          gh release create ${VERSION} --prerelease -t ${VERSION}
+
+  notify:
+    runs-on: ubuntu-latest
+    needs:
+      - prerelease
+    if: always() && failure() && github.event_name == 'push'
+    steps:
+      - name: Slack Notification
+        uses: Alfresco/alfresco-build-tools/.github/actions/send-slack-notification@v1.22.0
```
Comment (atchertchian) [inline .github/workflows/rc.yml]:
```suggestion
        uses: Alfresco/alfresco-build-tools/.github/actions/send-slack-notification@v1.23.0
```

File: .github/workflows/rc.yml (hunk 1 above)
Comment (igdianov) [reply .github/workflows/rc.yml]: yes

**Pull request merged**